bot/controllers/verify.js

```diff
@@ -110,9 +110,9 @@ class VerifyController extends BaseController {
       });
 
       util.log('Code', code, 3);
-      return msg.verify.verifyEmailMsgStart[LANGUAGE] + ` ${(timeoutInMiliseconds / 1000) / 60} ` + msg.verify.verifyEmailMsgEnd[LANGUAGE];
+      return msg.verify.verifyEmailMsgStart[lang] + ` ${(timeoutInMiliseconds / 1000) / 60} ` + msg.verify.verifyEmailMsgEnd[lang];
     } else {
-      return msg.verify.verifyEmailDenied[LANGUAGE];
+      return msg.verify.verifyEmailDenied[lang];
     }
   }
 }
```

Comment:
Make sure to default all languages to en. If none are set, or if an incorrect langauge is set, all messages appear as *underfined* in Discord.